src/force/CabanaPD_Force_PMB.hpp

```diff
@@ -300,6 +322,9 @@ class Force<ExecutionSpace, ForceModel<LinearPMB, Elastic>>
     using base_type::_half_neigh;
     ForceModel<LinearPMB, Elastic> _model;
 
+    using base_type::_energy_timer;
```

Comment:
@streeve In Lines 79-80, we have:
Timer _timer;
Timer _energy_timer;

Should we try to keep the same order consistent (first "_timer" then "_energy_timer") everywhere?

Reply:
This is automatically alphabetical order